integration_tests/views/main.view.js

```diff
@@ -0,0 +1,4 @@
+// import { $ } from 'protractor';
```

Comment:
Only files inside `src/` directory are processed with webpack, can you try moving integration_tests there?